example/crud.postman_collection.json

```diff
@@ -1276,13 +1252,50 @@
 			},
 			"response": []
 		},
+		{
+			"name": "get current elements at refa before updating them",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"return latest elements on refa\", function () {",
+							"    var jsonData = pm.response.json();",
+							"    pm.expect(jsonData.commitId).to.eql(pm.environment.get(\"commitXYRef\"))",
+							"",
+							"    pm.environment.set(\"commitXYRefElements\", pm.response.json());",
```

Comment:
using the entire json here to compare with later won't work since the return elements' _refId should reflect the context they're gotten in - getting the elements from a branch will have slightly different json even if they're the same elements (the _refId will be different)